cmd/bbolt/command_page_item.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"go.etcd.io/bbolt/internal/common"
+	"go.etcd.io/bbolt/internal/guts_cli"
+)
+
+const (
+	pageItemUsage = `usage: bolt page-item [options] PATH pageid itemid
+
+Additional options include:
+
+	--key-only
+		Print only the key
+	--value-only
+		Print only the value
+	--format
+		Output format. One of: ` + FORMAT_MODES + ` (default=auto)
+
+page-item prints a page item key and value.`
+)
+
+type pageItemOptions struct {
+	help      bool
```

Comment:
lgtm 👍🏽

Reply:
`help` is not necessary here. Cobra automatically add `--help` and `-h` for each sub-command.